src/main/kotlin/io/perfometer/runner/CoroutinesScenarioRunner.kt

```diff
@@ -1,8 +1,6 @@
 package io.perfometer.runner
 
-import io.perfometer.dsl.HttpStep
-import io.perfometer.dsl.PauseStep
-import io.perfometer.dsl.RequestStep
+import io.perfometer.dsl.*
```

Comment:
I think we should agree on common formatting options to avoid such changes in every commit.
I think I use default from intellij with only 2 changes (aligning method and constructor parameters when multiline)
I will make a commit with adding the rules to repository and we could discuss it.

Reply:
I think I also have defaults set up in Intellij :thinking: altough I'm switching between two PCs, maybe there's a difference in my IDE setup. Having a rules in the repo is the best option.

Reply:
I've just found an option in intellij to set code formatting to "Kotlin style guide" which seems to mean Kotlin Coding Conventions. It looks ok for me, if you agree I could go with that (it is available in code style -> kotlin -> set from... -> Kotlin style guide)

Reply:
I think this is exactly what I have enabled, I'll double-check it, and we can go with this setting.